class/exoscale-cloud-controller-manager.yml

```diff
@@ -1,5 +1,13 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: https
+        source: https://raw.githubusercontent.com/exoscale/exoscale-cloud-controller-manager/refs/tags/v${exoscale_cloud_controller_manager:images:exoscale_cloud_controller_manager:tag}/docs/examples/cloud-controller-manager-rbac.yml
```

Comment:
Yikes, that will be fun updating 😐

Reply:
depends on how often Exoscale will make "interesting" changes to the directory structure of their repo.